generate_sbom

```diff
@@ -935,6 +940,17 @@ sub spdx_encode_header {
   return $spdx;
 }
 
+sub spdx_encode_dist {
+  my ($dist) = @_;
+
+  return spdx_encode_pkg({
+    NAME => $dist->{id},
+    VERSION => $dist->{version_id},
+    spdx_id => sprintf('SPDXRef-OperatingSystem-%s', gen_pkg_id($dist)),
+    external_refs => 0
+  }, undef, undef, {});
+
```

Comment:
You need to have OPERATING-SYSTEM in there

Reply:
Added!